README.md

```diff
@@ -25,6 +27,47 @@ jobs:
       # ...
 ```
 
+
+### Using authn_id
+
+```yaml
+on: [push]
+
+jobs:
+  test:
+    # ...
+    permissions:
+      id-token: 'write'
+    steps:
+      # ...
+      - name: Import Secrets using CyberArk Conjur Secret Fetcher
+        uses: quincycheng/conjur-action@master
+        with:
+          url: ${{ secrets.CONJUR_URL }}
+          account: cyberarkdemo
+          authn_id: ${{ secrets.CONJUR_AUTHN_ID }}
+          secrets: db/sqlusername|sql_username;db/sql_password
+      # ...
+```
+
+Note:  JWT Authenticator is required at Conjur server.  You may wish to refer to [official doc](https://docs.cyberark.com/Product-Doc/OnlineHelp/AAM-DAP/Latest/en/Content/Operations/Services/cjr-authn-jwt.htm?tocpath=Integrations%7CJWT%20Authenticator%7C_____0) 
+The sample policy below validates GitHub repository & workflow
+
+1. [Sample authenticator policy](github-authn-jwt.yml)
+2. [Sample app id policy](github-app-id.yml)
+3. Sample secret values and commands.
+```
+conjur policy load -f ./policy/github-authn-jwt.yml -b root
+conjur policy load -f ./policy/github-app-id.yml -b root
+
+conjur variable set -i conjur/authn-jwt/github/issuer -v "https://token.actions.githubusercontent.com"
+conjur variable set -i conjur/authn-jwt/github/jwks-uri -v "https://token.actions.githubusercontent.com/.well-known/jwks"
+conjur variable set -i conjur/authn-jwt/github/token-app-property -v "workflow"
+conjur variable set -i conjur/authn-jwt/github/enforced-claims -v "workflow,repository"
+conjur variable set -i conjur/authn-jwt/github/identity-path -v "/github-apps"
+```
+
+
 ## Arguments
 
 ### Required
```

Comment:
This section should be split into two sub-sections `#### Host Identity` and `#### JWT Authentication` since there are different requirements for both.

Reply:
Sure. Updated.